pkgs/applications/networking/cluster/helm-dashboard/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
```

`installShellFiles` is not being used:

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```